sdk/nodejs/dagger/engine.ts

```diff
@@ -44,10 +44,22 @@ export class Engine {
     args.push("--port", `${this.config.Port}`);
 
     const serverProc = execa("dagger", args, {
-      stdio: "inherit",
+      stdin: "inherit",
+      stdout: "inherit",
+      stderr: "pipe",
       cwd: this.config.Workdir,
+    })
+    serverProc.catch(err => {
+      let error = (err as ExecaError);
```

Comment:
Would it be possible to use `err` directly instead of ExecaError ? Or is there a particular reason of using `ExecaError` ?